lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -728,6 +729,7 @@ void generateAndSortGuesses(InstContext &IC, Inst *LHS, SMTLIBSolver *Solver,
 
 typedef std::map<Inst*, std::vector<llvm::APInt>> InstConstList;
 
+#if 0
```

Comment:
can you explain why you don't want to just delete this code?

Reply:
This function of code has the support of input specialization while the new code implements the G by S, I want to do more comparisons between these two

Reply:
sounds good
in the future, if you plan to use the code again, protect it using "if (false)" instead of "#if 0" since then it has to pass the parser and typechecker